### PR TITLE
Add marquez-airflow version file to bumpversion 

### PR DESCRIPTION
**Pull request by mobuchowski**

### CODE REVIEW

**Comment by mobuchowski** [comment]
@wslulciuc updated

File: integrations/airflow/setup.cfg [hunk 1]
```diff
@@ -11,6 +11,10 @@ replace = version="{new_version}"
 search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
 
+[bumpversion:file:marquez_airflow/version.py]
```
Comment (wslulciuc) [inline integrations/airflow/setup.cfg:14]: We can now remove the reference to `marquez_airflow/version.py` in `setup.cfg`

Comment (mobuchowski) [reply integrations/airflow/setup.cfg]: Done

**Review by wslulciuc — APPROVED** [review]
LGTM 👍

**Pull request merged**